TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -560,17 +542,10 @@ class RouteOptionsViewController: UIViewController {
     }
 
     private func processRequest(result: Result<Response<RouteSectionsObject>>, requestURL: String, endPlace: Place) {
```

Comment:
Pretty sure we can take out this whole function now that we don't need to log the requestURL. Also can we just take out anything pertaining to the requestURL

Reply:
when you say "we can take out this whole function now", do you mean the processRequest function? Because I was under the impression that that is the function is actually loading our table results?

Reply:
yes but i'd rather keep the logic in `processRequest` connected with the actual call to `.observe`. We can either move the 5 lines above `processRequest(...)` into `processRequest` or just move the code from `processRequest` into here. Now that I think of it tho, might be better to just move those 5 lines into `processRequest`
<img width="834" alt="Screen Shot 2019-10-29 at 4 47 26 PM" src="https://user-images.githubusercontent.com/36868927/67807776-d98f4280-fa6b-11e9-96c9-4eeda37ac6d3.png">

Reply:
A note about the code above: I'm fairly sure that as of swift 4.2 the guard in that image is equivalent to

    guard let self = self else { return }

i.e. drop the tick marks.

Reply:
@Omarrasheed I moved the function into our processRequest function so now we make the getRoutes call inside of process request and we make one call to process request from searchforroutes!